txcache/selectionTracker.go

```diff
@@ -307,6 +322,7 @@ func (st *selectionTracker) removeFromTrackedBlocksNoLock(searchedBlock *tracked
 		"searched block prevHash", searchedBlock.prevHash,
 		"removed blocks", removedBlocks,
 	)
+	return nil
```

Comment:
Nitpicking / formatting - here, and in other place in this PR - can add newline before `return ...`.